runtime/databricks/automl_runtime/sklearn/base_datetime_transformer.py

```diff
@@ -117,5 +117,8 @@ def _generate_datetime_features(cls, X, include_timestamp=True):
 
         for holiday_calendar in [holidays.UnitedStates(), holidays.EuropeanCentralBank()]:
             features.append(X[col].apply(lambda datetime: datetime in holiday_calendar))
-
-        return pd.concat(features, axis=1)
+        ans = pd.concat(features, axis=1)
+        # Give non-string column names to avoid duplicated feature names, sklearn 1.0 does not work on duplicated
+        # feature nanmes.
+        ans.columns = range(len(ans.columns))
```

Comment:
Is this the only breakage to scikit-learn 1.0? Can we update https://github.com/databricks/automl/blob/64af351986dd6fb0798ef09129b8af0e6fbb92ae/runtime/environment.txt#L12 to ensure all the tests passed?

Reply:
Done. Also added a new test case.